app/qml/components/FieldRow.qml

```diff
@@ -21,22 +21,26 @@ Item {
     height: fieldDelegate.rowHeight
     width: fieldDelegate.width
     spacing: InputStyle.panelSpacing
-    property real itemSize: (parent.width - imageBtn.width - (3* row.spacing)) / 2
+    property real itemSize: (parent.width - imageBtn.width - (2* row.spacing)) / 2
 
     InputTextField {
       id: textField
       color: fieldDelegate.color
       text: AttributeName
       Layout.fillHeight: true
+      Layout.fillWidth: true
       Layout.preferredWidth: row.itemSize
 
-      onEditingFinished: AttributeName = text
+      onTextChanged: {
+        AttributeName = textField.text
+      }
```

Comment:
This did not quite help, I thought:

```suggestion
      onDisplayTextChanged: AttributeName = displayText
```
<img src="https://user-images.githubusercontent.com/22449698/110335148-d58e1000-8023-11eb-9ccc-1b309c05ade1.jpg" width=300>